### ISSUE TITLE
Ignores alignment of label text when content is mixed with HTML

### ISSUE THREAD

**Issue by solrankos** [schibsted/layout#46]
When i put bold on `world` the center alignment is ignored and the text is aligned to the left

```
<UIView
    width="100%"
    height="100%"
    backgroundColor="#fff">
    <UILabel
        width="100%"
        top="50% - height / 2"
        textAlignment="center"
        font="Courier bold 30"
        backgroundColor="red"
    >Hello <b>World</b></UILabel>
</UIView>
```

**Comment by nicklockwood** [comment]
Thanks, I'll look into it. In the meantime, using a `<center></center>` tag around your text might help.

**Comment by nicklockwood** [comment]
Fixed in 0.4.21
